### ISSUE TITLE
Multi-User

### ISSUE THREAD

**Issue by cwbsl** [LaraBug/larabug-app#110]
Firstly: Thanks a lot for this absolute great product! It's amazingly easy to use and so helpful!
Is it possible to add another developer to my projects? Or will this feature be built any time soon?

**Comment by cwbsl** [comment]
Yes, so I have a few projects in my dashboard and it all works fine.
Now I want to add another developer B to a project, and 2 other developers B & C to a third project so they can see the things I see.

**Comment by Cannonb4ll** [comment]
Understood, yeah teams is not available just yet - but would be a great feature to have. Feel free to send in a PR!

**Comment by cwbsl** [comment]
Ok no problem, thanks a lot! I will look into it!

**Comment by abhaywani114** [comment]
Can I contribute to add this feature?

**Comment by Cannonb4ll** [comment]
@abhaywani114 Yes, if you feel like sending in a PR that allows for team functionality, please!
